<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pdevmodbusui</class>
 <widget class="QDialog" name="Pdevmodbusui">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>347</width>
    <height>313</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>DevModBus</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../onewire.qrc">
    <normaloff>:/images/images/kfm_home.png</normaloff>:/images/images/kfm_home.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>332</width>
     <height>298</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="Config">
    <attribute name="title">
     <string>Config</string>
    </attribute>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="6" column="0" colspan="3">
      <widget class="QGroupBox" name="groupBoxWrite">
       <property name="title">
        <string>Write (check to enable control)</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLineEdit" name="Write_Value"/>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="hexWrite">
          <property name="text">
           <string>Hex</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="pushButtonWrite">
          <property name="text">
           <string>Write</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="labelCoef">
       <property name="text">
        <string>Coef :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="Address"/>
     </item>
     <item row="0" column="2">
      <widget class="QCheckBox" name="hexSlaveID">
       <property name="text">
        <string>Hex</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QComboBox" name="requestType"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Request type</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="Read_Mask_Enable">
       <property name="text">
        <string>Read Mask</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QCheckBox" name="hexAddress">
       <property name="text">
        <string>Hex</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QLineEdit" name="Read_Mask"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Address</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Slave ID</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="SlaveID"/>
     </item>
     <item row="4" column="0">
      <widget class="QCheckBox" name="Unsigned">
       <property name="text">
        <string>Unsigned</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="comboBoxResolution">
       <property name="currentText">
        <string>16 Bits</string>
       </property>
       <item>
        <property name="text">
         <string>16 Bits</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>32 Bits</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>64 Bits</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="M3">
    <attribute name="title">
     <string>M3</string>
    </attribute>
    <widget class="QCheckBox" name="MuxEnable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>341</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Use M3 Multiplexer</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Switch Values</string>
    </attribute>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2">
      <widget class="QCheckBox" name="MaxCheckBox">
       <property name="text">
        <string>Hex</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Max Value</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditMaxValue">
       <property name="text">
        <string>FFFF</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="QLabel" name="labelInfo">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../onewire.qrc"/>
 </resources>
 <connections/>
</ui>
